lib/delayed_job_worker_pool/worker_pool.rb

```diff
@@ -3,11 +3,14 @@
 
 module DelayedJobWorkerPool
   class WorkerPool
+
     SIGNALS = ['TERM', 'INT'].map(&:freeze).freeze
+    DEFAULT_WORKER_COUNT = 1
 
     def initialize(options = {})
       @options = options
       @worker_pids = []
+      @worker_pid_options = {}
```

Comment:
Can we combine the `@worker_pids` and `@worker_pid_options` data structures to make it easier to keep the information stored in them consistent?